.github/workflows/test-upstream.yaml

```diff
@@ -83,15 +84,15 @@ jobs:
           echo "upstream_repo=${{ inputs.upstream_repo }}" >> "$GITHUB_ENV"
 
       - name: checkout upstream
-        uses: actions/checkout@v4
+        uses: actions/checkout@v4.2
         with:
           repository: "${{ env.upstream_repo }}"
           path: linkml
           ref: "${{ env.upstream_branch }}"
           fetch-depth: 0
 
       - name: checkout linkml-runtime
-        uses: actions/checkout@v4
+        uses: actions/checkout@v4.2
         with:
           # don't specify repository like this or else we won't get pull request branches correctly
           # repository: linkml/linkml-runtime
```

Comment:
this looks like cruft - can it be removed?

Reply:
Done.